frontends/web/src/routes/buy/pocket.tsx

```diff
@@ -95,14 +96,23 @@ export const Pocket = ({ code }: TProps) => {
           String(message.withMessageSignature),
           code)
           .then(response => {
-            if (response.status === 'abort') {
-              // TODO notify to the widget with a message?
-              alertUser('Message signing aborted by the user');
-            } else {
+            if (response.status !== 'abort') {
               sendAddress(response.address, response.signature);
             }
           });
       }
+      if (message.type === V0MessageType.VerifyAddress) {
+        setVerifying(true);
+        verifyAddress('p2wpkh', message.bitcoinAddress, code)
```

Comment:
why all this code to select the right address based on script type in the backend when it's hardcoded to p2wpkh?

Reply:
Gosh! Forgot about that, now should be ok. I also defined native segwit as the default script type in case it is not defined in the request